src/scripts/main.js

```diff
@@ -1,3 +1,46 @@
 'use strict';
 
-// write code here
+const items = document.querySelectorAll('li');
+const list = document.querySelector('ul');
+
+function convertNumber(str) {
+  const newStr = str.slice(1);
+
+  return +newStr.split(',').join('');
+}
+
+function sortList(arr) {
+  const sortArr = Array.from(arr);
+
+  sortArr.sort((a, b) => {
+    const first = convertNumber(a.getAttribute('data-salary'));
+    const second = convertNumber(b.getAttribute('data-salary'));
+
+    return second > first ? 1 : second < first ? -1 : 0;
+  });
+
+  for (const item of sortArr) {
+    list.appendChild(item);
+  }
+
+  return sortArr;
+}
+
+function getEmployees(arr) {
+  const arrObjects = [];
+
+  for (const item of arr) {
```

Comment:
Schema for the object should looke like:
```js
{
  name, 
  position,
  salary,
  age
}
```

yours looks like this:
<img width="134" alt="image" src="https://github.com/mate-academy/js_task_parse_list_DOM/assets/126275137/da143ada-eb65-43dc-8b1c-8b8a23fd4a27">


Fix object schema